pkg/antctl/raw/multicluster/clusterset.go

```diff
@@ -0,0 +1,199 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package multicluster
+
+import (
+	"context"
+	"fmt"
+	"sort"
+	"strings"
+
+	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	multiclusterv1alpha1 "antrea.io/antrea/multicluster/apis/multicluster/v1alpha1"
+	"antrea.io/antrea/pkg/antctl/raw"
+)
+
+var cmdClusterSet *cobra.Command
+
+type clusterSetOptions struct {
+	namespace     string
+	outputFormat  string
+	allNamespaces bool
+}
+
+var optionsClusterSet *clusterSetOptions
+
+var clusterSetExamples = strings.Trim(`
+Gel all cluster sets in default namesapce
```

Comment:
Please change all "namespace" to "Namespace"; cluster set to "ClusterSet", in all messages, logs, comments, etc.. We capitalize the first letter of resource types.